exporter/sematextexporter/factory.go

```diff
@@ -28,7 +36,24 @@ func NewFactory() exporter.Factory {
 }
 
 func createDefaultConfig() component.Config {
-	cfg := &Config{}
+	cfg := &Config{
+		ClientConfig: confighttp.ClientConfig{
+			Timeout: 5 * time.Second,
+			Headers: map[string]configopaque.String{
+				"User-Agent": "OpenTelemetry -> Sematext",
```

Comment:
FYI this is how the OTLP and OTLP/HTTP exporter [set](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.120.0/exporter/otlphttpexporter/otlp.go#L70-L71) their user agent string. This resolves to `OpenTelemetry Collector Contrib/0.119.0 (linux/amd64)` when built in Contrib binary. You're free to set the user agent value to whatever you wish though.